### ISSUE TITLE
Error: Unable to resolve module `./Libraries/Components/AccessibilityInfo/AccessibilityInfo` from `node_modules/react-native/index.js`

### ISSUE THREAD

**Issue by Abhishek12345679** [microsoft/react-native-macos#575]
### Environment

1. `react-native -v`: 0.63.2
3. `npm ls react-native-macos`: ^0.62.0-0
4. `node -v`: v12.14.1
5. `npm -v`: 6.14.7
6. `yarn --version`: 1.22.4
7. `xcodebuild -version`: Xcode 11.3.1 Build version 11C505

### Steps to Reproduce

I followed the basic steps provided in react-native-macos docs

1. npx react-native init <projectName> 
2. cd <projectName>
3. npx react-native-macos-init
4. npx react-native start
5. npx react-native run-macos

### Expected Behavior

It should have launched properly , like a basic untouched react-native-macos app

### Actual Behavior

![Screen Shot 2020-09-03 at 3 11 17 PM](https://user-images.githubusercontent.com/24722640/92099423-1a788600-edf8-11ea-88b2-8a5d0e7d78c4.jpg)

### Reproducible Demo

You can just follow the steps above.

**Comment by Abhishek12345679** [comment]
when is it supposed to go live ?

**Comment by acoates-ms** [comment]
Can you try this with react-native-macos 0.62.11?

**Comment by Abhishek12345679** [comment]
let me check  @acotes-ms

**Comment by Abhishek12345679** [comment]
It works but not with the command from the react-native-docs (npx react-native run-macos)

Instead it works when we start the metro server and app separately as -

yarn start:macos
npx react-native run-macos
